src/resources/extensions/quarto/video/video.lua

```diff
@@ -152,12 +153,32 @@ local videoJSBuilder = function(params)
   result.id = id
   return result
 end
+
+local dropboxBuilder = function(params)
+  if not (params and params.src) then return nil end
+  local src = params.src
+  local isDropbox = function()
+    return string.find(src, 'https://www.dropbox.com') and checkEndMatch(src, '?raw=1')
+  end
+
+  if not isDropbox() then return nil end
+
+  local result = {}
+
+  local SNIPPET = [[<iframe data-external="1" src="{src}"{width}{height} allowfullscreen="" title="{title}" allow="encrypted-media"></iframe>]]
```

Comment:
Please consider adding this to the unit tests.

Reply:
I'm confused too. 

I've just tried again pasting my video code into the `index.qmd` of a newly created book project, and `quarto preview` in version 1.2.280 still gives me `No video source specified for video shortcode` and refuses to continue the preview. The version built from my PR works fine. 

I don't know anything about Lua or videoJS, at all, so I'm at a loss.

Reply:
@tobydriscoll You need to test against 1.3 prereleases.